---
title: System.IdentityModel.IdentityModelAsyncCallbackThrewException
ms.custom: 
ms.date: 03/30/2017
ms.prod: .net-framework
ms.reviewer: 
ms.suite: 
ms.technology: dotnet-clr
ms.tgt_pltfrm: 
ms.topic: article
ms.assetid: e4fc924f-3b9e-44f5-97e4-b3a4bb855d89
caps.latest.revision: "4"
author: dotnet-bot
ms.author: dotnetcontent
manager: wpickett
ms.openlocfilehash: ce02d785c76b3e8168ee5ceb19841fe649bc23d1
ms.sourcegitcommit: ce279f2d7fe2220e6ea0a25a8a7a5370ddf8d9f0
ms.translationtype: MT
ms.contentlocale: cs-CZ
ms.lasthandoff: 12/02/2017
---
# <a name="systemidentitymodelidentitymodelasynccallbackthrewexception"></a>System.IdentityModel.IdentityModelAsyncCallbackThrewException
System.IdentityModel.IdentityModelAsyncCallbackThrewException  
  
## <a name="description"></a>Popis  
 Asynchronní zpětné volání došlo k výjimce.  
  
## <a name="see-also"></a>Viz také  
 [Trasování](../../../../../docs/framework/wcf/diagnostics/tracing/index.md)  
 [Řešení potíží s vaší aplikace pomocí trasování](../../../../../docs/framework/wcf/diagnostics/tracing/using-tracing-to-troubleshoot-your-application.md)  
 [Správa a Diagnostika](../../../../../docs/framework/wcf/diagnostics/index.md)
